common-ui/src/main/res/layout/row_two_line_item.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?><!--
-  ~ Copyright (c) 2021 DuckDuckGo
+  ~ Copyright (c) 2018 DuckDuckGo
```

Comment:
Back in 2018 :smile:

Reply:
Not sure if it's related to this change, but I noticed the downloads are now missing the leading / trailing icon.
![downloads_miss_leading_trailing_icons](https://user-images.githubusercontent.com/7963079/191962449-6251c2d6-57e6-49c2-ad54-b65790c520e1.png)